README.md

```diff
@@ -345,6 +345,32 @@ Deploy package built with `assemble_rpm` to RPM repository.
 | <a id="deploy_rpm-target"></a>target |  <code>assemble_rpm</code> target to deploy   | <a href="https://bazel.build/concepts/labels">Label</a> | optional | <code>None</code> |
 
 
+<a id="doxygen_docs"></a>
+
+## doxygen_docs
+
+<pre>
+doxygen_docs(<a href="#doxygen_docs-name">name</a>, <a href="#doxygen_docs-desc">desc</a>, <a href="#doxygen_docs-main_page_md">main_page_md</a>, <a href="#doxygen_docs-project_name">project_name</a>, <a href="#doxygen_docs-sources">sources</a>, <a href="#doxygen_docs-strip_prefix">strip_prefix</a>, <a href="#doxygen_docs-version">version</a>)
+</pre>
+
+
+        Creates HTML documentation for C++ projects using Doxygen.
```

Comment:
The docs should note that Doxygen must be installed externally - this is not a hermitic rule.